hyperjump

```diff
@@ -149,19 +148,16 @@ function jj() {
 		local param
 		for param in ${@:2}
 		do
-			if [ ! -z "$param" -a "$param" != "" ]
-			then
+			if [[ ! -z "$param" -a "$param" != "" ]]; then
```

Comment:
It works in zsh, hmm, I should try zsh one of these days. I am still running the good old bash :)